tensorflow/compiler/mlir/hlo/include/mlir-hlo-c/Attributes.h

```diff
@@ -175,105 +173,105 @@ mlirMhloConvDimensionNumbersGetOutputSpatialDimensionsElem(MlirAttribute attr,
 //
 // Creates a new ComparisonDirection attribute with the given
 // 'direction' string parameter.
-MLIR_CAPI_EXPORTED MlirAttribute mlirMhloComparisonDirectionAttrGet(
-    MlirContext ctx, const std::string &direction);
+MLIR_CAPI_EXPORTED MlirAttribute
+mlirMhloComparisonDirectionAttrGet(MlirContext ctx, const char *direction);
 
 // Returns true if the given attribute is a ComparisonDirection attribute.
 MLIR_CAPI_EXPORTED bool mlirMhloAttributeIsAComparisonDirectionAttr(
     MlirAttribute attr);
 
 // Returns the direction string associated with ComparisonDirection attribute.
-MLIR_CAPI_EXPORTED std::string mlirMhloComparisonDirectionAttrGetDirection(
-    MlirAttribute attr);
+MLIR_CAPI_EXPORTED const char *
```

Comment:
Can we use mlirStringRef here? That would be more in line with upstream conventions.